__tests__/components/ui/Avatar.test.tsx

```diff
@@ -0,0 +1,71 @@
+import Avatar from "@/components/ui/Avatar";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userIcon, userProfile } from "../../../__mocks__/userProfile";
+
+describe("Avatar", () => {
+  it("Should render the avatar", () => {
+    render(<Avatar />);
+    const avatar = screen.getByTestId("avatar");
+
+    expect(avatar).toBeInTheDocument();
+  });
+
+  it("Should render an avatar link", () => {
+    render(<Avatar />);
+    const avatar = screen.getByTestId("avatar");
+    expect(avatar.tagName).toBe("A");
+  });
+
+  it("Should render an avatar div box", () => {
+    render(<Avatar useLink={false} />);
```

Comment:
The name of the text should change. the purpose for us is to check that link doesn't get include in case it's not enable